src/CommonLib/utility.cxx

```diff
@@ -627,3 +627,30 @@ UTILITY::LogEvent(
         fwprintf(stderr, L"ERROR: %s\n", pstrMsg);
     }
 }
+
+VOID
+UTILITY::LogEventF(
```

Comment:
LogFailureEvent? I'm not familiar with the F suffix.

Reply:
[F]ormat? Like in printf

Reply:
Ahh. Fine with me then.